src/Datasource/Paginator.php

```diff
@@ -220,7 +220,10 @@ public function paginate($object, array $params = [], array $settings = [])
         $this->_pagingParams = [$alias => $paging];
 
         if ($requestedPage > $page) {
-            throw new PageOutOfBoundsException([$requestedPage]);
+            throw new PageOutOfBoundsException([
+                'requestedPage' => $requestedPage,
+                'pagingParams' => $this->_pagingParams
```

Comment:
We should either add the alias here too, or narrow down the paging params to only the subset of this alias IMO.
Otherwise it can be difficult to get to the needed params here.